Cargo.toml

```diff
@@ -160,7 +160,7 @@ js-sys = "0.3.67"
 wasm-bindgen = "0.2.87"
 wasm-bindgen-futures = "0.4.40"
 wasm-bindgen-test = "0.3"
-web-sys = "0.3.67"
+web-sys = "=0.3.67"
```

Comment:
It would be good to have a comment here explaining why we are pinning to a specific version.